### PR TITLE
Feat/CORL-2213 remove story live update config

### PR DESCRIPTION
**Pull request by marcushaddon**
## What does this PR do?
This PR removes the option to toggle live updates via the config panel in the stream.

## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## How do I test this PR?
While logged in as an admin, navigate to a stream's Configure tab, and see that there is no longer the option to toggle live updates for that particular story (they can still be toggled site wide).

## How do we deploy this PR?
This shouldn't require any special considerations.

### CODE REVIEW

**Comment by marcushaddon** [comment]
@nick-funk @tessalt I just removed LiveUpdateConfig component from the stream config component, without deleting source for for LiveUpdateConfig. Let me know if I should delete all of it or if we have some way of identifying dead code.

File: src/core/client/stream/tabs/Configure/Configure.tsx [hunk 1]
```diff
@@ -33,8 +33,6 @@ const Configure: FunctionComponent<Props> = (props) => {
       <AddMessageContainer story={props.story} />
       <QAConfigContainer story={props.story} settings={props.settings} />
       <HorizontalRule />
-      <LiveUpdatesConfigContainer story={props.story} />
-      <HorizontalRule />
```
Comment (nick-funk) [inline src/core/client/stream/tabs/Configure/Configure.tsx]: Are there translations that might need to be removed from `stream.ftl` as well?

**Review by nick-funk — CHANGES_REQUESTED** [review]

**Review by alinhle — CHANGES_REQUESTED** [review]
Hey @marcushaddon, you can remove the files entirely. We can always take it back from this PR. Also there are quite a few unused translations. Can you find and remove them? Unfortunately we have do this all by hand at the moment.

**Review by nick-funk — APPROVED** [review]

File: src/locales/en-US/stream.ftl [hunk 2]
```diff
@@ -694,26 +694,11 @@ configure-premodLink-commentsContainingLinks =
...
-configure-liveUpdates-description =
-configure-enableLiveUpdates-description =
-  When enabled, the comments will be updated instantly as new comments and
-  replies are submitted, instead of requiring a page refresh. You can
-  disable this in the unusual situation of an article getting so much
-  traffic that the comments are loading slowly.
-configure-enableLiveUpdates-enable = Enable
-
-configure-disableLiveUpdates-title = Disable live updates
 configure-disableLiveUpdates-description =
```
Comment (alinhle) [inline src/locales/en-US/stream.ftl]: Here is one left to be removed :-)

File: src/core/client/stream/tabs/Configure/LiveUpdatesConfig/LiveUpdatesConfigContainer.tsx [hunk 3]
```diff
@@ -1,161 +0,0 @@
...
-  story: graphql`
-    fragment LiveUpdatesConfigContainer_story on Story {
-      id
-      settings {
-        live {
-          configurable
-          enabled
-        }
-      }
-    }
```
Comment (alinhle) [inline src/core/client/stream/tabs/Configure/LiveUpdatesConfig/LiveUpdatesConfigContainer.tsx:157]: The fragment `LiveUpdatesConfigContainer_story` is still being used, that's why we got all the build errors.

**Review by alinhle — APPROVED** [review]

**Review by nick-funk — APPROVED** [review]
Approving again

**Pull request merged**